### PR TITLE
Feature/main page styling i

### PR DESCRIPTION
**Pull request by Matt-Roden**
# Description drawing

- Added some base styling and structure to the main page of application
- Added fonts and images folders into project

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by hayleyw7 — APPROVED** [review]
Looks great! Very pretty!

File: src/components/App/App.css [hunk 1]
```diff
@@ -1,38 +1,6 @@
...
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+  /* border: solid green 5px; */
   display: flex;
   flex-direction: column;
-  align-items: center;
```
Comment (joanafbrito) [inline src/components/App/App.css:21]: Like the cleaning! Thank you

File: src/components/App/App.js [hunk 2]
```diff
@@ -74,13 +75,16 @@ class App extends Component {
     const text = 'Loading...';
 
     return(
-      <div> Welcome to PoKeDeX, are you ready to catch your Pokemon?
+      <div className='App'>
```
Comment (joanafbrito) [inline src/components/App/App.js:78]: We probably can change this div for a` section` Element

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+.custom-shape-divider-bottom-1631589819 svg {
+    position: relative;
+    display: block;
+    width: calc(100% + 1.3px);
+    height: 100px;
+}
+
+.custom-shape-divider-bottom-1631589819 .shape-fill {
+    fill: #FFFFFF;
+}
```
Comment (joanafbrito) [inline src/components/Header/Header.css:55]: If we can would be cool to to some refactoring in the css / so we don't have to do that only by the end ( ordered list of name and values)

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+      <nav className='nav'>
+        <ul className='nav-links'>
+          <li className='how-to'>How To</li>
+        </ul>
+      </nav>
+
+      <img className='header-image' src='Images/ash1.png' alt='ash-ketchum'/>
+
+      <div class="custom-shape-divider-bottom-1631589819">
```
Comment (joanafbrito) [inline src/components/Header/Header.js:17]: great use of divs here

**Review by joanafbrito — APPROVED** [review]
It looks amazing
My only observations is for us to try to refactor the css while doing that

**Pull request merged**